widgets/multiple-choice-list/index.js

```diff
@@ -1,24 +1,60 @@
 var React = require('react');
+var cx = require('classnames');
 
 var utils = require('../../lib/widgetUtils.js');
-var defaultTemplate = require('./template.html');
 
-function multipleChoiceList({container = null, facetName = null, template = defaultTemplate}) {
+var defaultTemplate = `<label>
+  <input type="checkbox" value="{{name}}" {{#isRefined}}checked{{/isRefined}} />{{name}} <span>{{count}}</span>
+</label>`;
+
```

Comment:
Why did you put the template inline here? Seems easier to edit when in its own `html` file.